wagtailio/static/sass/components/_logos.scss

```diff
@@ -4,11 +4,11 @@
     position: relative;
 
     &__heading {
-        margin-bottom: 20px;
+        margin-bottom: 80px;
```

Comment:
We might want to reduce this, and maybe increase the margin below the logos instead? Feels a bit weird that the bottom margin of the heading is larger (80px) than the bottom margin between the logos and the next heading (50px), which is a separate group.

<img width="1066" alt="image" src="https://user-images.githubusercontent.com/6379424/201958929-269fbf7e-2041-4ba2-af2f-c2805a89456e.png">


<img width="1018" alt="image" src="https://user-images.githubusercontent.com/6379424/201958767-a5a5cfe8-9fe6-4f6a-b38a-d199859cb899.png">

Reply:
Submitted a commit that sets the margin to `60px` for all views. Please let me know if there's anything else I should fix.